collections/_development/code-reviews.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title:  "Code Reviews"
+area: development
+---
+
+# Code Reviews
+
+Code reviews are an essential part of the development process and must be carried out on all code regardless of where the change originates. The process should also be carried out with the appropriate rigour and attention. Some things to keep in mind during the code review process:
+
+* Give reviews your full attention
+* Limit the amount of time you spend reviewing in any one sitting
+* If necessary check the code out and test it locally
```

Comment:
Personally I think you should always get the branch and run locally. A further step is to make sure it functionally works as per the requirement before reviewing.

Reply:
I agree that in the main you should be getting out the code locally and I'll update this to reflect that. However there will always be edge cases (just updating the README springs to mind) so that also needs reflecting in the Principles